collector-bitmovin-player/src/test/java/com/bitmovin/analytics/bitmovin/player/utils/AdBreakMapperTest.kt

```diff
@@ -30,35 +31,49 @@ class AdBreakMapperTest {
     @Before
     fun setup(){
         MockitoAnnotations.initMocks(this)
+        `when`(adConfiguration.replaceContentDuration).thenReturn(10.0)
+
         `when`(adBreak.id).thenReturn("id")
+        `when`(adBreak.scheduleTime).thenReturn(10.0)
+
         `when`(imaAdBreak.id).thenReturn("id")
         `when`(imaAdBreak.tag).thenReturn(AdTag("", AdTagType.VMAP))
+        `when`(imaAdBreak.position).thenReturn("pre")
     }
 
     @Test
-    fun test__FromPlayerAdConifurationWithAdConfiguration(){
-        `when`(adConfiguration.replaceContentDuration).thenReturn(10.0)
+    fun FromPlayerAdConfigurationWithAdConfigurationShouldSetValuesOfAdConfiguration(){
         val collectorAdBreak = adBreakMapper.FromPlayerAdConfiguration(adConfiguration)
 
-        Assert.assertNotNull(collectorAdBreak.replaceContentDuration)
-        Assert.assertNull(collectorAdBreak.scheduleTime)
+        assertThat(collectorAdBreak.replaceContentDuration).isNotNull()
     }
 
     @Test
-    fun test__FromPlayerAdConfigurationWithAdBreak() {
-        `when`(adBreak.scheduleTime).thenReturn(10.0)
+    fun FromPlayerAdConfigurationWithAdConfigurationShouldNotSetValuesOfAdBreak(){
+        val collectorAdBreak = adBreakMapper.FromPlayerAdConfiguration(adConfiguration)
+
+        assertThat(collectorAdBreak.scheduleTime).isNull()
+    }
+
+    @Test
+    fun FromPlayerAdConfigurationWithAdBreakShouldSetValuesOfAdBreak() {
         val collectorAdBreak = adBreakMapper.FromPlayerAdConfiguration(adBreak)
 
-        Assert.assertNotNull(collectorAdBreak.scheduleTime)
-        Assert.assertNull(collectorAdBreak.position)
+        assertThat(collectorAdBreak.scheduleTime).isNotNull()
```

Comment:
Since you are setting it to a specific value at the `setup` it would be better to also check that value here. The test does what you intend - but it doesn't really check if the value is also correctly set - and if we were to change the data type to a not-nullable int (which defaults to 0) this test would not fail